wire-ios-data-model/Source/Model/Conversation/TypingUsers/NSManagedObjectContext+TypingUsers.swift

```diff
@@ -23,7 +23,7 @@ extension NSManagedObjectContext {
 
     private static let TypingUsersKey = "ZMTypingUsers"
 
-    var typingUsers: TypingUsers? {
+    public var typingUsers: TypingUsers? {
```

Comment:
if TypingUsers just contain NSManagedObjectIDs I don't see why we force it to be on a specific context

Reply:
yes I agree, it doesn't make sense we don't need to use a specific context

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/2194/commits/75a2e7c46a506ea4b18825e2cfa19c33f8a5e470